gulpfile.js

```diff
@@ -127,25 +131,26 @@ const getTextDimensions = () =>
       size: Number.parseInt(value, 10),
   }));
 
-const getTextStyles = fontWeight => {
+const getTextStyles = () => {
```

Comment:
we now get all the styles as they are back, rather than deciding what's default/emphasized/heavy on the client side